Content.Server/GameObjects/Components/Doors/AirlockComponent.cs

```diff
@@ -52,6 +54,42 @@ private bool PowerWiresPulsed
             }
         }
 
+        private bool _boltsDown;
+        private bool BoltsDown
+        {
+            get => _boltsDown;
+            set
+            {
+                _boltsDown = value;
+                EntitySystem.Get<AudioSystem>()
+                .PlayFromEntity(value ? "/Audio/machines/boltsdown.ogg" : "/Audio/machines/boltsup.ogg",
+                    Owner);
+                UpdateWiresStatus();
+                UpdateBoltLightStatus();
+            }
+        }
+
+        private bool _boltLights = true;
+        private bool BoltLightsOn
+        {
+            get => _boltLights;
+            set
+            {
+                _boltLights = value;
+                UpdateWiresStatus();
+                UpdateBoltLightStatus();
+            }
+        }
```

Comment:
The one pitfall I can see from this design is that it has to be updated manually. The state of the bolt light should always be decided by the bolt light wire and the bolt state (and maybe door power), but this might allow them to get out of sync.

Reply:
Can you access the AirlockComponent from the Client AirlockVisualizer?